src/components/svg/IconEdit.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
```

Comment:
Actually this file needs to be auto-generated. I added some instructions in the README.
Copy the 'icon-edit.svg' to './assets' and run `yarn generate-assets`.

Reply:
Oh, forgot about that!